packages/manager/apps/web-domains/src/domain/data/api/domainResources.ts

```diff
@@ -15,6 +15,30 @@ export const getDomainResource = async (
 export const getDomainAnycastOption = async (
   serviceName: string,
 ): Promise<TDomainOption> => {
-  const { data } = await v6.get(`/domain/${serviceName}/option/${OptionEnum.DNS_ANYCAST}`);
+  const { data } = await v6.get(
+    `/domain/${serviceName}/option/${OptionEnum.DNS_ANYCAST}`,
+  );
+  return data;
+};
+
+/**
+ *  : Update DNS configuration of resource
```

Comment:
```suggestion
 *  : Update a domain resource
```
If this function has to be used to update all the fields of the resource, update the comment; if this function should only be used to update the DNS (which mean that other ones should be defined to update each other field of the resource), rename the function (`updateDomainDnsConfiguration` for instance)